src/stream/src/executor/sync_kv_log_store.rs

```diff
@@ -225,8 +428,11 @@ impl<S: LocalStateStore> WriteFuture<S> {
         stream: BoxedMessageStream,
         write_state: LogStoreWriteState<S>,
     ) -> Self {
+        let instant = Instant::now() + duration;
+        tracing::trace!(?instant, ?duration, "write_future_pause");
```

Comment:
```suggestion
        tracing::trace!(now = ?Instant::now(), ?duration, "write_future_pause");
```

We may lazy evaluate the `Instant::now()`.